test/validation/validation-test.js

```diff
@@ -0,0 +1,55 @@
+var config = require("../../src/config");
+var site = require("../../src/site");
+var links = require("../../src/links");
+var shell = require("shelljs")
+var path = require("path");
+
+var fs = require("fs-extra");
+
+var assert = require('assert');
+
+let res = {};
+
+describe('Link Validation', () => {
+
+    describe('Build Site', () => {
+        var conf = config.load("./test/validation/_config.yml");
```

Comment:
We can move this into a shared test setup method once we work with other site generation test cases...